src/js/environment/browser-version.js

```diff
@@ -0,0 +1,53 @@
+export function browserVersion(browserEnvironment, agent) {
+    let version;
+    let major;
+    let minor;
+    let ix;
+
+    if (browserEnvironment.chrome) {
+        if (agent.indexOf('CriOS') !== -1) {
+            version = agent.substring(agent.indexOf('CriOS') + 6);
+        } else {
+            version = agent.substring(agent.indexOf('Chrome') + 7);
+        }
+    } else if (browserEnvironment.safari) {
+        // Safari sets its version after the "Version" string in the agent
+        version = agent.substring(agent.indexOf('Version') + 8);
+    } else if (browserEnvironment.firefox) {
+        version = agent.substring(agent.indexOf('Firefox') + 8);
+    } else if (browserEnvironment.edge) {
+        // Search for "Edge/" specifically, since some rare UA's contain the word "Edge"
```

Comment:
You mentioned searching for `"Edge/"` in the comment but you're checking `"Edge"` here. Is this intended?